pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -49,6 +51,61 @@ func NewServerOptions(useEmulation bool) *ServerOptions {
 type Launcher struct {
 	domainManager virtwrap.DomainManager
 	useEmulation  bool
+	notifier      *eventsclient.Notifier
+}
+
+func (i *Launcher) HandleDomainEvent(_ *cmdv1.EmptyRequest, stream cmdv1.Cmd_HandleDomainEventServer) error {
```

Comment:
where does this "stream" object come from? Is this some automatic GRPC thing ?

Reply:
Yes. You can see it in to `proto` file here: https://github.com/kubevirt/kubevirt/pull/3311/files#diff-e0be460ab0130ecceeebe771b1c66440R24.